### PR TITLE
Add Semantic Kernel Pivot to Open AI Quick Starts

### PR DESCRIPTION
**Pull request by JakeRadMSFT**
## Summary

This PR is to add SK (Semantic Kernel) pivots for the code sections in the AI Quick Starts.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/quickstarts/get-started-azure-openai.md](https://github.com/dotnet/docs/blob/186e3b3e137c82d74e52afa6e7f32be87dbc14b9/docs/ai/quickstarts/get-started-azure-openai.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code.](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/get-started-azure-openai?branch=pr-en-us-40205) |
| [docs/ai/quickstarts/quickstart-ai-chat-with-data.md](https://github.com/dotnet/docs/blob/186e3b3e137c82d74e52afa6e7f32be87dbc14b9/docs/ai/quickstarts/quickstart-ai-chat-with-data.md) | [Get insight about your data from an .NET Azure AI chat app](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-ai-chat-with-data?branch=pr-en-us-40205) |
| [docs/ai/quickstarts/quickstart-azure-openai-tool.md](https://github.com/dotnet/docs/blob/186e3b3e137c82d74e52afa6e7f32be87dbc14b9/docs/ai/quickstarts/quickstart-azure-openai-tool.md) | [Extend Azure AI using Tools and execute a local Function with .NET](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-azure-openai-tool?branch=pr-en-us-40205) |
| [docs/ai/quickstarts/quickstart-openai-generate-images.md](https://github.com/dotnet/docs/blob/186e3b3e137c82d74e52afa6e7f32be87dbc14b9/docs/ai/quickstarts/quickstart-openai-generate-images.md) | [Quickstart - Generate images using Azure AI with .NET](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-openai-generate-images?branch=pr-en-us-40205) |
| [docs/ai/quickstarts/quickstart-openai-summarize-text.md](https://github.com/dotnet/docs/blob/186e3b3e137c82d74e52afa6e7f32be87dbc14b9/docs/ai/quickstarts/quickstart-openai-summarize-text.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code to summarize text.](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-openai-summarize-text?branch=pr-en-us-40205) |

### CODE REVIEW

File: docs/ai/quickstarts/quickstart-openai-generate-images.md [hunk 1]
```diff
@@ -26,6 +27,54 @@ Get started with the .NET Azure OpenAI SDK by creating a simple .NET 8 console c
...
+// == Retrieve the local secrets saved during the Azure deployment ==========
+var config = new ConfigurationBuilder()
+    .AddUserSecrets<Program>()
+    .Build();
+
+var config = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
+string endpoint = config["AZURE_OPENAI_ENDPOINT"];
+string deployment = config["AZURE_OPENAI_GPT_NAME"];
+string key = config["AZURE_OPENAI_KEY"];
+```
```
Comment (aaronpowell) [inline docs/ai/quickstarts/quickstart-openai-generate-images.md:56]: This looks out of order, it should be before the `AzureOpenAITextToImageService` is created shouldn't it?

File: docs/ai/quickstarts/get-started-azure-openai.md [hunk 2]
```diff
@@ -26,6 +27,82 @@ Get started with the .NET Azure OpenAI SDK by creating a simple .NET 8 console c
...
+
+The entire application is contained within the **Program.cs** file. The first several lines of code loads up secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.
+
+```csharp
+// == Retrieve the local secrets saved during the Azure deployment ==========
+var config = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
+string endpoint = config["AZURE_OPENAI_ENDPOINT"];
+string deployment = config["AZURE_OPENAI_GPT_NAME"];
+string key = config["AZURE_OPENAI_KEY"];
+```
```
Comment (aaronpowell) [inline docs/ai/quickstarts/get-started-azure-openai.md:53]: Shouldn't this be before the `AzureOpenAIChatCompletionService` creation?

**Review by aaronpowell — CHANGES_REQUESTED** [review]
I think the order of setting up the secrets and using the Chat/Image services is backwards, as you set that up before getting the keys, so if you copy the code in as it stands it won't compile.

Given that there's repeated steps across each of them (setting up the secrets), should we pull that out as an include so we can have that referenced into each article rather than copy/pasted?

File: docs/ai/quickstarts/get-started-azure-openai.md (hunk 2 above)
Comment (JakeRadMSFT) [reply docs/ai/quickstarts/get-started-azure-openai.md:53]: Using the other thread for the convo.
https://github.com/dotnet/docs/pull/40205#discussion_r1538391742

File: docs/ai/quickstarts/quickstart-openai-generate-images.md (hunk 1 above)
Comment (JakeRadMSFT) [reply docs/ai/quickstarts/quickstart-openai-generate-images.md:56]: I agree - I was copying the Azure Open AI SDK pattern/ordering but maybe we should change both?

Comment (JakeRadMSFT) [reply docs/ai/quickstarts/quickstart-openai-generate-images.md:56]: @FBoucher  thoughts?

Comment (aaronpowell) [reply docs/ai/quickstarts/quickstart-openai-generate-images.md:56]: Ah, I didn't realise that was how the AOAI SDK was structured.

Comment (JakeRadMSFT) [reply docs/ai/quickstarts/quickstart-openai-generate-images.md:70]: Sounds good  - I'll tackle this in a follow up PR.

Comment (JakeRadMSFT) [reply docs/ai/quickstarts/quickstart-openai-generate-images.md:70]: https://github.com/dotnet/docs/issues/40218

**Review by fboucher — APPROVED** [review]

File: docs/ai/quickstarts/get-started-azure-openai.md [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
+:::zone target="docs" pivot="azure-openai-sdk"
+<!-- markdownlint-enable MD044 -->
+
 Get started with the .NET Azure OpenAI SDK by creating a simple .NET 8 console chat application. The application will run locally and use the OpenAI `gpt-35-turbo` model deployed into an Azure OpenAI account. Follow these steps to provision Azure OpenAI and learn how to use the .NET Azure OpenAI SDK.
 
+:::zone-end
+
 [!INCLUDE [download-alert](includes/prerequisites-and-azure-deploy.md)]
 
 ## Trying HikerAI sample
```
[the remark this answers is not in the document]
Comment (JakeRadMSFT) [reply docs/ai/quickstarts/get-started-azure-openai.md:33]: Yep! That's resolved in the next PR!

[the remark this answers is not in the document]
Comment (JakeRadMSFT) [reply docs/ai/quickstarts/get-started-azure-openai.md:33]: https://github.com/JakeRadMSFT/dotnet_docs/pull/1/files

**Review by luisquintanilla — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**